mypy/checkexpr.py

```diff
@@ -182,6 +182,16 @@ def visit_call_expr(self, e: CallExpr, allow_none_return: bool = False) -> Type:
                 e.callee.node.typeddict_type is not None:
             return self.check_typeddict_call(e.callee.node.typeddict_type,
                                              e.arg_kinds, e.arg_names, e.args, e)
+        if isinstance(e.callee, NameExpr) and e.callee.name in ('isinstance', 'issubclass'):
+            for typ in mypy.checker.flatten(e.args[1]):
+                try:
+                    if (isinstance(typ, IndexExpr)
+                            and isinstance(typ.analyzed, (TypeApplication, TypeAliasExpr))
+                            or isinstance(typ, NameExpr)
+                            and self.chk.lookup_qualified(typ.name).kind == nodes.TYPE_ALIAS):
+                        self.msg.type_arguments_not_allowed(e)
+                except KeyError:
```

Comment:
Not sure. Here's where it happens when running `mypy mypy`:

    # stubgenc.py
    all_bases = obj.mro()
    # ...
    for base in all_bases:
        if not any(issubclass(b, base) for b in bases): # builtins.KeyError: 'Failed lookup: bases'
            bases.append(base)
and here:

    # testextensions.py    
    def assertIsSubclass(self, cls, class_or_tuple, msg=None):
        if not issubclass(cls, class_or_tuple): # failed lookup

and here:

        Emp = TypedDict('Emp', {'name': str, 'id': int})
        with self.assertRaises(TypeError):
            isinstance({}, Emp)

In all cases, the names are typed as `Any`, but why weren't they added to the symbol table, and how there were no messages about using undefined names?